modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -140,14 +140,7 @@ public DDMFormEvaluatorEvaluateResponse evaluate() {
 			rule -> {
 				evaluateDDMFormRule(rule);
 
-				_ddmFormFieldsPropertyChanges.forEach(
-					(key, value) -> {
-						if (_ddmFormEvaluatorEvaluateRequest.isViewMode() &&
-							!isFieldVisible(key)) {
-
-							value.put("value", StringPool.BLANK);
-						}
-					});
+				_resetInvisibleFieldValue();
```

Comment:
@linolaoj, do we need to keep this method here or can we invoke it as the other methods, like `verifyFieldsMarkedAsRequired`, `validateFields` and `_localizeNumericDDMFormFieldValues`?

Reply:
@natocesarrego , we need to keep this method here because the value needs to be reset between rules evaluation. The next rule needs to take action with the value already reset.

Reply:
Thanks for your clarification, @linolaoj. 🙂